src/univariate/continuous/noncentralchisq.jl

```diff
@@ -1,3 +1,24 @@
+"""
+    NoncentralChisq(ν, λ)
+
+The *Noncentral Chi-squared distribution* (typically written χ²) with `ν` degrees of freedom and non-centrality parameter `λ` has the probability density function
```

Comment:
I think both "noncentral" and "chi" should be lowercase. Also the parenthesized note about the notation is probably unnecessary. I'd also remove the hyphen in "non-centrality" for consistency with "noncentral."